src/components/login.js

```diff
@@ -1,17 +1,82 @@
-import React from "react";
+import React, { useEffect, useState, useRef } from "react";
 import { Link } from "react-router-dom";
+import * as yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import getAllUsers from "../common/get-all-users";
+import getUserByAccountNumber from "../common/get-user-act-no";
+import { useNavigate } from "react-router-dom";
+
 export const Login = () => {
+  const [option, setOption] = useState([]);
+  const [inputValue, setInputValue] = useState();
+  const navigate = useNavigate();
+  const selectRef = useRef(null);
+
+  const handleInputChange = (event) => {
+    setInputValue(event.target.value);
+  };
+
+  useEffect(() => {
+    const accountNames = getAllUsers();
+    const optionElem = accountNames.map((user, index) => (
```

Comment:
No need to do this, however, this is not the right way to update state data. Remove this useEffect function.